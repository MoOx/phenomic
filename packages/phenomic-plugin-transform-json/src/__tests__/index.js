import transformJSON from "../index.js"

it("should transform json as... json", () => {
  const plugin = transformJSON()
  expect(typeof plugin.transform === "function").toBe(true)
  expect(
    plugin.transform &&
      plugin.transform({
        file: {
          name: "file.json",
          fullpath: "/test/file.json",
          exists: true,
          type: "wat",
        },
        contents: new Buffer(
          `{
      "test": "a",
      "test2": "b",
      "body": "I am supposed to be huge"
    }`,
        ),
      }),
  ).toMatchSnapshot()
})
