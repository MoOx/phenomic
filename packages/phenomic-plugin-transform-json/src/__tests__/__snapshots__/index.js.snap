// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should transform json as... json 1`] = `
Object {
  "data": Object {
    "body": "I am supposed to be huge",
    "path": "file",
    "test": "a",
    "test2": "b",
  },
  "partial": Object {
    "path": "file",
    "test": "a",
    "test2": "b",
  },
}
`;
